lib/segment/src/entry/entry_point.rs

```diff
@@ -60,6 +61,15 @@ impl OperationError {
     }
 }
 
+pub fn check_optimization_stopped(stopped: &AtomicBool) -> OperationResult<()> {
```

Comment:
`check_optimization_stopped` is used in the segment builder. I propose renaming this function

Reply:
But the function in segment builder is only used during optimization.

Reply:
ok, that's a bad pattern, I agree

Reply:
renamed to `check_process_stopped`